### DISCUSSION TITLE
https://thomashallam.github.io/posts/2022-11-11-6-months-lead-user-researcher/toms-6-month-reflection.html

### DISCUSSION THREAD

**Discussion by giscus** [thomashallam/thomashallam.github.io#5] [General]
# https://thomashallam.github.io/posts/2022-11-11-6-months-lead-user-researcher/toms-6-month-reflection.html

https://thomashallam.github.io/posts/2022-11-11-6-months-lead-user-researcher/toms-6-month-reflection.html

**Comment by thomashallam** [comment]
test

**Comment by thomashallam** [comment]
CHATGPT

"The author is reflecting on their experience 6 months into their new role as a UCD-Lead/Lead User Researcher. They mention that the role and projects have been more complex than they expected, but that there have been some positives such as working on new projects, having a strong team, and focusing on quality over speed. They also mention some challenges such as coordinating change across different groups and teams, feeling disconnected from the "frontline," and difficulty starting the UCD Maturity project. They mention they have been working on the UCD Maturity project by hosting a community of practice, leading workshops, and providing guidance to teams. They mention the challenges they've faced in this project including lack of engagement from "non-delivery" teams, scepticism about UCD, and competing agendas. They also mention they're trying to provide more evidence of the benefits of UCD Maturity workshops to get more teams involved."

**Comment by thomashallam** [comment]
CHATGPT take 2

"The author is reflecting on their experiences in their new role as a UCD-Lead/Lead User Researcher. They mention that the role and projects have been more complex than they expected, but also highlight some positive aspects such as working with a great team and finding greenfield projects to work on. They mention that coordinating change across different groups and teams can take a lot of time, and that they have had to learn to prioritize and say no to requests. They also mention that they have been dropped into a politically complex NHS program, and that it was stressful but ended up working out well. They also mention that they have been working on a project called UCD Maturity, which has been difficult to get started but the community of practice and facilitators are supportive. They plan to run an evaluation study to understand the benefits of the workshops and to get more teams on board.
